configure.ac

```diff
@@ -15,7 +16,8 @@ AC_PATH_PROG([XSLTPROC], [xsltproc])
 PKG_PROG_PKG_CONFIG()
 
 PKG_CHECK_MODULES(LIBOAUTH, oauth)
-PKG_CHECK_MODULES(LIBPCRE, libpcre)
+#PKG_CHECK_MODULES(LIBPCRE2, libpcre2)
+PKG_CHECK_MODULES(PCRE2, [libpcre2-8 libpcre2-32])
```

Comment:
Just curious: why `-8` and `-32`?\
On my Debian/unstable system I can see:
```
/usr/lib/x86_64-linux-gnu/pkgconfig/libpcre2-16.pc
/usr/lib/x86_64-linux-gnu/pkgconfig/libpcre2-32.pc
/usr/lib/x86_64-linux-gnu/pkgconfig/libpcre2-8.pc
/usr/lib/x86_64-linux-gnu/pkgconfig/libpcre2-posix.pc
```

Reply:
I've removed the -32